src/datepicker/datepicker.ts

```diff
@@ -139,6 +149,12 @@ export class NgbDatepicker implements OnChanges,
    */
   @Input() startDate: {year: number, month: number};
 
+  /**
+   * An event fired when navigation happens and currently displayed month changes.
+   * The payload is the new month we're navigating to.
+   */
+  @Output() navigate = new EventEmitter<{year: number, month: number}>();
```

Comment:
Don't we want previous and next month so people could figure out direction of navigation? Just asking since changing it in the future would be a bc.